backend/lookup/api/visitorsUtils.py

```diff
@@ -0,0 +1,62 @@
+"""
+a module for creating a singleton class of visitors data.
+"""
+
+from lookup.models import Visitor
+from lookup.api.serializers import VisitorSerializer
+
+class Singleton(type):
+    """
+    a singleton class. use this as a metaclass to create a class.
+    """
+    _instances = {}
+
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
+        return cls._instances[cls]
+
+class VisitorsData(metaclass=Singleton): # use Singleton Class as a metaclass
```

Comment:
implemented a singleton class(because of the reason describe in description of this PR)